### DISCUSSION TITLE
updateContact

### DISCUSSION THREAD

**Discussion by rajeevn1** [AsamK/signal-cli#676] [Q&A] [answered]
signal-cli updateContact -e 28800 <number>
signal-cli: error: argument -n/--name is required

why is name required when I just want to update the expiration

**Comment by AsamK** [comment, marked as answer]
A leftover from the time when expiration hadn't been implemented and only the name could be changed.
Will be fixed in the next version (8dced20b0f448ddb003c2c20aaf71a2f7f751ce0)

**Reply by rajeevn1** [reply]
Thank you.

**Reply by AsamK** [reply]
released in 0.8.5

**Comment by rajeevn1** [comment]
signal-cli updateContact -n "First Last" -e 28800 <num>
error: This command doesn't work on linked devices.

Will I be able to change expiration (without changing name) on linked devices in the next version.

**Reply by AsamK** [reply]
yes, changing only the expiration will also work on linked devices.
